qiskit/visualization/pulse_v2/drawing_objects.py

```diff
@@ -0,0 +1,251 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+r"""
+Drawing object IRs for pulse drawer.
+
+Drawing IRs play two important roles:
```

Comment:
```suggestion
Drawing objects play two important roles:
```
(and elsewhere).  I agree its an IR in an abstract sense, but this terminology AFAIK isn't typically used in plotting and could be confusing.